audiorecorder.1

```diff
@@ -1,21 +1,21 @@
-.TH audiorecorder 1 "github.com/amiaopensource/audiorecorder" "2017\-06\-03" "AMIA Open Source"
+.TH audiorecorder 1 "github.com/amiaopensource/audiorecorder" "2017\-10\-18" "AMIA Open Source"
 .SH NAME
 audiorecorder - Tool for calibration and recording of analog audio sources
 .SH SYNOPSIS
 .B audiorecorder [-e | -p | -m | -b | -h]
 .SH OPTIONS
 .TP
 .B -e
-edit
+edit config
 .TP
 .B -p
-passthrough
+passthrough mode
 .TP
 .B -m
-metadata
+edit metadata for BWF insertion
```

Comment:
Maybe this would be a good chance to make this more explicit? Such as "edit metadata to be used in BWF creation"?